### PR TITLE
Deploy Rails App to Ubuntu Server 16.04 with Nginx, Puma, Capistrano

### PR DESCRIPTION
**Pull request by quanghm-0027**
## Deploy Rails App to Ubuntu Server 16.04 with
* Nginx
* Puma
* Capistrano

## Issue
https://edu-redmine.sun-asterisk.vn/issues/15572

## Link
http://42.113.206.204:8888/

### CODE REVIEW

**Comment by quanghm-0027** [comment]
ready

**Comment by quanghm-0027** [comment]
ready

**Comment by quanghm-0027** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,10 @@ group :development do
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "web-console", ">= 3.3.0"
+  gem 'capistrano', '~> 3.10.1'
+  gem 'capistrano-rails', '~> 1.3.1' 
+  gem 'capistrano-rbenv', '~> 2.1'
+  gem 'capistrano3-puma', '~> 3.1', '>= 3.1.1'
```
Comment (vanvtt-0952) [inline Gemfile]: nháy đôi hì

**Comment by quanghm-0027** [comment]
ready

**Pull request merged**